pyproject.toml

```diff
@@ -0,0 +1,54 @@
+[tool.poetry]
+name = "quartical"
+version = "0.1.10"
+description = "Fast and flexible calibration suite for radio interferometer data."
+repository = "https://github.com/ratt-ru/QuartiCal"
+documentation = "https://quartical.readthedocs.io"
+authors = ["Jonathan Kenyon <jonathan.simon.kenyon@gmail.com>"]
+license = "MIT"
+readme = "README.md"
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Intended Audience :: Science/Research",
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: POSIX :: Linux",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Topic :: Scientific/Engineering :: Astronomy"
+]
+include = [
+    "quartical/config/argument_schema.yaml",
+    "quartical/config/gain_schema.yaml",
+    "quartical/stimela_cabs.yaml"
+]
+
+[tool.poetry.dependencies]
+python = "^3.8"
+tbump = "^6.10.0"
+columnar = "^1.4.1"
+"ruamel.yaml" = "^0.17.26"
+dask-ms = {extras = ["s3", "xarray", "zarr"], version = "^0.2.9"}
+codex-africanus = {extras = ["dask", "scipy", "astropy", "python-casacore"], version = "^0.3.4"}
+astro-tigger-lsm = "^1.7.2"
+loguru = "^0.7.0"
+dask = {extras = ["diagnostics"], version = "^2022.9.1"}
+distributed = "^2022.9.1"
+requests = "^2.31.0"
+pytest = "^7.3.1"
```

Comment:
Not critical at all but perhaps a `testing` extra containing `tbump`, `requests` and `pytest` might be nice so that one can do a `pip install Quartial[testing]`

One can  do this by declaring them as optional dependencies like so:

https://github.com/ratt-ru/dask-ms/blob/master/pyproject.toml#L16C22-L21

and then creating a testing group containing those optional dependencies  like so

https://github.com/ratt-ru/dask-ms/blob/master/pyproject.toml#L26-L32

Reply:
Ah thanks, I will consider it!